### DISCUSSION TITLE
"agc workflow status" report disappearance

### DISCUSSION THREAD

**Discussion by staskh** [aws/amazon-genomics-cli#485] [Q&A]
Hi,

I'm evaluating AGC ( AWS Genomics CLI) fro our large scale project and just paid attention  that all reports (like "agc workflow status") disappear after about 24-48 h. Is it intended feature or I have some misconfiguration?
How can I configure AGC to keep all reports/outputs/statuses until explicit cleanup?

Regards
Stas

**Comment by markjschreiber** [comment]
Hi Stas,

I believe that the workflow status is limited by the retention of AWS Batch tasks which was 48 hours but has recently increased to one week. Workflow statuses will also be lost when a context is destroyed. 

Hope this helps,
Mark
